### PR TITLE
Fixed Broken link readme.md #132

### PR DESCRIPTION
**Pull request by bhartik021**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We welcome contributions on the docs site!
 3. Hack on your changes.
 4. Write a meaningful commit message (and sign your commit). Review the [commit message guidelines](https://github.com/sigstore/sigstore-website/blob/main/CONTRIBUTORS.md#commit-message-guidelines).
 5. Ensure that CI passes, if it fails, fix the failures. If you are making many commits into one PR, please [squash your commits](https://github.com/sigstore/sigstore-website/blob/main/CONTRIBUTORS.md#squash-commits).
-6. Every pull request requires a review from the [core sigstore-website team](https://github.com/orgs/github.com/sigstore/teams/codeowners-sigstore-website) before merging.
+6. Every pull request requires a review from the [core sigstore-website team](https://github.com/orgs/sigstore/people) before merging.
```
Comment (haydentherapper) [inline README.md:18]: This is too broad. It looks like there isn't a group, so I'd just omit this list

**Review by haydentherapper — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ltagliaferri) [reply README.md:18]: We can just remove the link here

Comment (bhartik021) [reply README.md]: fixed

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```
[the remark this answers is not in the document]
Comment (haydentherapper) [reply LICENSE:189]: Bump on reverting this

**Comment by git-maniesh** [comment]
Is the issue still open .... want to work on this issue and be a part GSoD

**Comment by ltagliaferri** [comment]
> Is the issue still open .... want to work on this issue and be a part GSoD

You were asked to revert one of the changes and haven't

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Please revert the copyright line in the license

**Comment by ltagliaferri** [comment]
Closing due to inactivity.

**Pull request closed without merging**